services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
```

Comment:
The current filtering logic in `get_movies` applies separate filters for `genres_ids` and `actors_ids`, which can lead to unintended results if both filters are used simultaneously. Consider combining these filters using `queryset = queryset.filter(genres__id__in=genres_ids, actors__id__in=actors_ids)` to ensure both conditions are applied correctly.